lib/streamlit/elements/spinner.py

```diff
@@ -34,9 +34,13 @@ def spinner(
     Parameters
     ----------
     text : str
-        The text to display next to the spinner. Defaults to "In progress...".
+        The text to display next to the spinner. This defaults to
```

Comment:
We could add here that it also supports markdown.

Reply:
Btw. Markdown is also supported but not mentioned in the following parameters:
- `st.success`, `st.info`, `st.warning`, `st.error` body
- `st.form_submit_button` label
- `st.pills`, `st.segmented_control` options & format_func return
- `help` parameter wherever it is supported.